### ISSUE TITLE
A11y_code.visualstudio.com_Docs Setup_Network_ScreenReader: Screen reader is announcing control type twice for link.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode-docs#2780]
“[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com)
Windows Version: Windows10
Browser Version:
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 1.3.1

**Repro Steps:**
Navigate to the URL: [https://code.visualstudio.com/docs/setup/network](https://code.visualstudio.com/docs/setup/network)
follow below steps:
1.Launch the application URL: [https://code.visualstudio.com](https://code.visualstudio.com)
2.Navigate to "Docs" link.
3.From left navigation, select "Setup".
4.Navigate to "Network" link and activate it.
5.Navigate to the main content.
6.Start the screen reader.
7.Navigate to the “Edit” link and to “Network Connections in Visual Studio Code” heading element.

**Actual:**
While navigating through “Edit” link, NVDA is announcing it as "Link Link Edit"
i.e. Control type is announced twice. 

This issue has only been found in Chrome+NVDA. 

**Expected:**
Control type should be announced only once and properly. 

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Screen reader user will get confused while navigating through link.

**MAS Reference:**
MAS 1.3.1 - 
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

**Attachment for Reference:**
[Code.visualstudio.com_Docs Setup_Network_ScreenReader announcing control type twice for link..pptx](https://github.com/microsoft/vscode-docs/files/3314110/Code.visualstudio.com_Docs.Setup_Network_ScreenReader.announcing.control.type.twice.for.link.pptx)

**Comment by RohitJadhav-TCS** [comment]
Similar Issue is observed in following screens:
1) Themes : [https://code.visualstudio.com/docs/getstarted/themes](https://code.visualstudio.com/docs/getstarted/themes)
2) Continuous Integration :[https://code.visualstudio.com/api/working-with-extensions/continuous-integration](https://code.visualstudio.com/api/working-with-extensions/continuous-integration)
3) Display Language :[https://code.visualstudio.com/docs/getstarted/locales
](https://code.visualstudio.com/docs/getstarted/locales)
4) Angular Tutorial :[https://code.visualstudio.com/docs/nodejs/angular-tutorial
](https://code.visualstudio.com/docs/nodejs/angular-tutorial)
5) Java : [https://code.visualstudio.com/docs/languages/java](https://code.visualstudio.com/docs/languages/java)
